app/(connected)/(with-sidebar)/(with-chat)/mes-jeunes/[idJeune]/informations/InformationsPage.tsx

```diff
@@ -0,0 +1,358 @@
+          textColor='content_color'
+          iconName={IconName.Error}
+        />
+        <TileIndicateur
+          valeur={actions?.terminees.toString() ?? '-'}
+          label={actions?.terminees !== 1 ? 'Terminées' : 'Terminée'}
+          bgColor='accent_2_lighten'
+          textColor='accent_2'
+          iconName={IconName.CheckCircleFill}
+        />
+        <TileIndicateur
+          valeur={actions?.aEcheance.toString() ?? '-'}
+          label='Échéance cette semaine'
+          bgColor='primary_lighten'
+          textColor='primary_darken'
+        />
+      </ul>
+      {idJeune && <LienVersActions idJeune={idJeune} pathPrefix={pathPrefix} />}
+    </div>
+  )
+}
+
+function IndicateursRendezvous({
+  rendezVous,
+}: Partial<Pick<IndicateursSemaine, 'rendezVous'>>) {
+  return (
+    <div className='border border-solid rounded-base w-full mt-6 p-4 border-grey_100'>
+      <h3 className='text-m-bold text-content_color mb-4'>Les événements</h3>
+      <ul className='flex'>
+        <TileIndicateur
+          valeur={rendezVous?.toString() ?? '-'}
+          label='Cette semaine'
+          bgColor='primary_lighten'
+          textColor='primary_darken'
+        />
+      </ul>
+    </div>
+  )
+}
+interface IndicateursOffresProps
+  extends Partial<Pick<IndicateursSemaine, 'offres' | 'favoris'>> {
+  idJeune?: string
+  pathPrefix: string
+}
+
+function IndicateursOffres({
+  offres,
+  favoris,
+  idJeune,
+  pathPrefix,
+}: IndicateursOffresProps) {
+  return (
+    <div className='border border-solid rounded-base w-full mt-6 p-4 border-grey_100'>
+      <h3 className='text-m-bold text-content_color mb-4'>Les offres</h3>
+      <ul className='flex flex-wrap gap-2'>
+        <TileIndicateur
+          valeur={offres?.consultees.toString() ?? '-'}
+          label={
+            offres?.consultees !== 1 ? 'Offres consultées' : 'Offre consultée'
+          }
+          bgColor='primary_lighten'
+          textColor='primary_darken'
+        />
+        <TileIndicateur
+          valeur={favoris?.offresSauvegardees.toString() ?? '-'}
+          label={
+            favoris?.offresSauvegardees !== 1
+              ? 'Favoris ajoutés'
+              : 'Favori ajouté'
+          }
+          bgColor='primary_lighten'
+          textColor='primary_darken'
+        />
+        <TileIndicateur
+          valeur={offres?.partagees.toString() ?? '-'}
+          label={
+            offres?.partagees !== 1 ? 'Offres partagées' : 'Offre partagée'
+          }
+          bgColor='primary_lighten'
+          textColor='primary_darken'
+        />
+        <TileIndicateur
+          valeur={favoris?.recherchesSauvegardees.toString() ?? '-'}
+          label={
+            favoris?.recherchesSauvegardees !== 1
+              ? 'Recherches sauvegardées'
+              : 'Recherche sauvegardée'
+          }
+          bgColor='primary_lighten'
+          textColor='primary_darken'
+        />
+      </ul>
+      {idJeune && (
```

Comment:
l'affichage du lien des favoris est conditionné par l'autorisation du jeune, dans les métadonneesFavoris
  metadonneesFavoris: { offres, recherches, autoriseLePartage } -> autoriseLePartage
il faut ajouter la condition.
```suggestion
      {idJeune && autoriseLePartage (
```
tu peux voir dans BlocFavoris